### DISCUSSION TITLE
Spoilers

### DISCUSSION THREAD

**Discussion by ARR4NN** [remarkjs/remark#1134] [Q&A] [answered]
How can I do spoilers with a button like this?
![chrome_o377lcXAfA](https://user-images.githubusercontent.com/70060472/225427666-fa8b75a0-d8bc-4bf9-9b31-96dcdec8b5ca.gif)
is their a plugin?

**Comment by ChristianMurphy** [comment, marked as answer]
Heya @ARR4NN!
Your question is a bit vague, please see asking quality questions, for some ideas on how to frame questions to make it easier or more effective for others to help you: https://github.com/remarkjs/.github/blob/main/support.md

Spoilers are not a native feature of HTML or Markdown.
You could solve this from the HTML side with a custom element https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements
Or from the markdown side with a directive https://github.com/remarkjs/remark-directive

Then building you custom spoiler logic using those.
